### ISSUE TITLE
Add API for managing cookies

### ISSUE THREAD

**Issue by alechenninger** [darcy-framework/darcy-web#21]
Not necessarily final, but we need something like this:

New API in Browser like...
```java
browser.cookies().add(new Cookie(/*....*/));
browser.cookies().get("cookie_name");
browser.cookies().deleteAll()
// etc.etc.etc.
```

So that means you need two new classes in addition to Browser API changes:
`CookieManager` (what `browser.cookies()` returns)
And a `Cookie` class itself, which should basically look very similar to the one in WebDriver (if licenses are compatible we should consider actually use that one exactly and attributing it to the Selenium team in the license header but I'm not sure of all the legal ramifications of this).

Then you'll need to implement this in darcy-webdriver.

Do this work in a new version 0.2.1-SNAPSHOT so we don't break darcy-webdriver 0.2.0-SNAPSHOT. Same for darcy-webdriver version (it should be the next in line which I think should also be 0.2.1-SNAPSHOT).

**Comment by abocz** [comment]
Selenium uses Apache, and Apache v2 and GPL v3 are compatible, see here:
 https://www.gnu.org/philosophy/license-list.html#GPLCompatibleLicenses

Apache's v2 license: http://www.apache.org/licenses/LICENSE-2.0

tl;dr version from Apache's FAQs:
http://www.apache.org/foundation/license-faq.html#WhatDoesItMEAN
